content/support.md

```diff
@@ -6,7 +6,11 @@ description: Learn about support options available in Galaxy
 
 All Galaxy customers can submit support cases from within Galaxy and receive Basic online support for deployment-related questions. There is no guaranteed response SLA for Basic support but questions are typically answered within 1-2 business days.  
 
-You can submit tickets on [galaxy.meteor.com/support](https://galaxy.meteor.com/support)
+Galaxy support is dedicated to resolving Galaxy-specific issues and troubleshooting cases where Galaxy's performance may be at fault. While we strive for "best effort" with regard to questions pertaining to Meteor, they are not an officially supported Galaxy subject area. 
+
+If you would like expert assistance with your Meteor question, consider a subscription to Galaxy Developer Support. You are also welcome to post your issue to the forums or to GitHub, depending on the situation.
```

Comment:
"Galaxy Developer Support" should be "Meteor Developer Support"
Link to https://www.meteor.com/developer-support